x-pack/plugins/security_solution/public/management/cypress/e2e/automated_response_actions/form.cy.ts

```diff
@@ -18,7 +18,7 @@ import { cleanupRule, generateRandomStringName, loadRule } from '../../tasks/api
 import { RESPONSE_ACTION_TYPES } from '../../../../../common/api/detection_engine';
 import { login, ROLE } from '../../tasks/login';
 
-describe('Form', { tags: '@ess' }, () => {
+describe('Form', { tags: ['@ess', '@serverless', '@brokenInServerless'] }, () => {
```

Comment:
Is the test really broken in Serverless or you wanna just skip it for now? `@skipInServerless` tag could be a better choice, see [here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/README.md#how-to-add-a-new-cypress-test)